engines/cine/cine.cpp

```diff
@@ -321,6 +323,10 @@ void CineEngine::showSplashScreen() {
 			}
 		}
 	}
+	
+	Common::Keymapper *keymapper = _eventMan->getKeymapper();
+	keymapper->getKeymap("mouse-shortcuts")->setEnabled(true);
```

Comment:
It also looks like this splash screen only shows on specific games. So "mouse-shortcuts" now only enables if this is shown. I recommend disabling "intro-shortcuts" by default, enabling it at the top of this method.
You may be able to leave "mouse-shortcuts" always enabled, as mapping a key to multiple actions should dispatch all those actions when pressed.